docs/publishing/index.md

```diff
@@ -0,0 +1,125 @@
+
+<style>
+.content__default ul {list-style:none;padding:0;}
+</style>
+
+### Do: 👍
+
+- ✅ [Do publish latest standard EcmaScript](#do-publish-latest-standard-ecmascript)
+- ✅ [Do publish standard es modules](#do-publish-standard-es-modules)
+- ✅ [Do include `"main": "index.js"` and `"module": "index.js"` in your `package.json`](#do-include-main-indexjs-and-module-indexjs-in-your-packagejson)
+- ✅ [Do export element classes](#do-export-element-classes)
+- ✅ [Do export side effects separately](#do-export-side-effects-separately)
+- ✅ [Do keep side effects out of your index.js](#do-keep-side-effects-out-of-your-indexjs)
+- ✅ [Do import 3rd party node modules with "bare" import specifiers](#do-import-3rd-party-node-modules-with-bare-import-specifiers)
+- ✅ [Do include file extensions in import specifiers](#do-include-file-extensions-in-import-specifiers)
+- _Optional_: [use `type: "module"` when possible](#optional-use-type-module)
+- _Optional_: [include typings; e.g. `*.d.ts` files](#optional-include-typings)
+- _Optional_: [include source maps](#optional-include-source-maps)
+
+### Don't: 👎
+
+- ❌ [Do not optimize](#do-not-optimize)
+- ❌ [Do not bundle](#do-not-bundle)
+- ❌ [Do not minify](#do-not-minify)
+- ❌ [Do not use `.mjs` file extensions](#do-not-use-mjs-file-extensions)
+- ❌ [Do not import polyfills](#do-not-import-polyfills)
+
+### General concept: Building is an application level concern
+
+Your users will always have the most intimate knowledge of how and where thier code will be used to their users. Give them the greatest amount of flexibility in preparing their code for those conditions by publishing your package with the agreement that building is an application level concern. Any decision that you can make in support of your users having an easier time when and if they decide to bundle is a good decision to make when preparing your package for publication. Every "do" and every "don't" that is included herein will help you in doing just that.
+
+## Do publish latest standard EcmaScript
+
+Write standard EcmaScript from the start, and publishing standard EcmaScrip will come naturally. However, if you choose to write write your component in TypeScript or with various emerging specifications or APIs, be sure to convert your code to standard EcmaScript before publication. Transpiling your published javascript below a reasonably modern level (e.g. [TC39 Stage 4](https://github.com/tc39/proposals/blob/master/finished-proposals.md) and/or available cross-browser) will increase the likelihood that verbose or duplicate code is included in any final production delivery of pages featuring your package. Take this into account in cases where you use newer specifications that may require special care when delivering to various browsers. If for any reason you choose to ship non-transpiled code featuring newer specification(s), be sure to document this reality thoroughly so that your users can prepare to leverage your package appropriately.
```

Comment:
```suggestion
Write standard EcmaScript from the start, and publishing standard EcmaScrip will come naturally. However, if you choose to write your component in TypeScript or with various emerging specifications or APIs, be sure to convert your code to standard EcmaScript before publication. Transpiling your published javascript below a reasonably modern level (e.g. [TC39 Stage 4](https://github.com/tc39/proposals/blob/master/finished-proposals.md) and/or available cross-browser) will increase the likelihood that verbose or duplicate code is included in any final production delivery of pages featuring your package. Take this into account in cases where you use newer specifications that may require special care when delivering to various browsers. If for any reason you choose to ship non-transpiled code featuring newer specification(s), be sure to document this reality thoroughly so that your users can prepare to leverage your package appropriately.
```

> be sure to convert your code to standard EcmaScript before publication. Transpiling your published javascript below a reasonably modern level (e.g. [TC39 Stage 4](https://github.com/tc39/proposals/blob/master/finished-proposals.md) and/or available cross-browser) will increase the likelihood that verbose or duplicate code is included in any final production delivery of pages featuring your package.

I think we can be more clear here; this reads to me as "Totally transpile your code!". I think we should put more emphasis on the stage 4 'baseline' of transpilation, and mention explicitly to _not_ transpile to es5 for example

> If for any reason you choose to ship non-transpiled code

Im still not sure if we should even say this; the recommendation is to transpile those

Reply:
It _may_ be that we want to lean away from the hedge and just say "never transpile your code"? 🦆🦆🦆🦆🦆🦆

I'll take another pass.

Reply:
Imo we should be clear about this: 
If you use non-standard syntax; transpile that down. That is the recommendation. If we mention 'if you do not choose to do this', to me it feels like we're giving them an escape hatch: "oh, I guess some people don't transpile those features down? then maybe its fine for me too 🤷 ". I just think we shouldnt mention that at all, because the recommendation is; transpile non-standard syntax.